src/content/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer.mdx

```diff
@@ -257,6 +258,35 @@ At the working space you can see:
       src="/images/queries-nrql_screenshot-crop_metrics-events-share-export.webp"
     />
   </Collapser>
+
+    <Collapser
+    className="freq-link"
+    id="share-export"
+    title="Forecast"
+  >
+
+    <Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+    If you have opted for On-demand forecasting, you can add forecast to line and area charts, which uses historical data to project future trends. 
```

Comment:
Should it be On-Demand Forecasting, or should "On-demand forecasting" be otherwise set out as the name of a feature?

I'd also say "...you can add FORECAST to...", since it's a NRQL keyword, or change it to "you can add forecasting..." or "you can add a forecast..."